oss_audit/java/oss_audit.bzl

```diff
@@ -187,7 +194,7 @@ def _oss_audit_impl(ctx):
                 artifact_id,
                 name,
                 srcjar_url if srcjar_url else "",
-                version,
+                version
```

Comment:
What is the reason for removing this `,`? buildifier is generating warnings from it.

Reply:
accident. fixed.